src/NuGet.Clients/NuGet.PackageManagement.UI/Resources/Brushes.cs

```diff
@@ -149,7 +156,21 @@ public static class Brushes
  
         public static object ButtonBorderFocusedStyleBrushKey { get; private set; } = SystemColors.HighlightTextBrushKey; 
  
-        public static object ButtonFocusedStyleBrushKey { get; private set; } = SystemColors.HighlightTextBrushKey; 
+        public static object ButtonFocusedStyleBrushKey { get; private set; } = SystemColors.HighlightTextBrushKey;
+
+        public static object TabInactiveBrushKey { get; private set; } = SystemColors.InactiveCaptionBrushKey;
```

Comment:
TabInactiveBrushKey isn't used anywhere

Reply:
Intended for storing the color brushes for tabs that are neither active nor hovered. Since there's a brush for general UI text that also covers this case for tabs, I'll remove this unused brush.